app/models/template_invocation.rb

```diff
@@ -0,0 +1,9 @@
+class TemplateInvocation < ActiveRecord::Base
+
+  belongs_to :template, :class_name => "JobTemplate", :foreign_key => "template_id"
+  belongs_to :job_invocation, :inverse_of => :template_invocations
+  has_many :template_invocation_input_values
+
+  alias_attribute :input_values, :template_invocation_input_values
```

Comment:
just out of curiosity, is there a benefit over specifying custom `:class name` in `has_many` few lines above? I like `input_values` and I think we don't need the longer variant